packages/client/hmi-client/src/components/workflow/simulate-ciemss-operation.ts

```diff
@@ -0,0 +1,30 @@
+import { TimeSpan } from '@/types/Types';
+import { Operation, WorkflowOperationTypes } from '@/types/workflow';
+import { ChartConfig } from '@/types/SimulateConfig';
+
+export interface SimulateOperationState {
+	chartConfigs: ChartConfig[];
+	currentTimespan: TimeSpan;
+}
```

Comment:
I think we need to rename this, cause the same named interface is being exported and it might make things a tad confusing.

Reply:
this type is a duplicate of the julia version, do we want to use the same type for both or have one type for ciemss and julia each?

Reply:
im going to have one type for each, we can change it down the line if we want